src/features/rbac/components/ProjectMembersTable.tsx

```diff
@@ -119,8 +119,8 @@ export function ProjectMembersTable({ projectId }: { projectId: string }) {
                     {hasDeleteAccess ? (
                       <TableCell>
                         <Button
-                          variant="ghost"
-                          size="xs"
+                          variant="destructive"
```

Comment:
I just checked that in the UI and think it is very dominant in the UI right now. What do you think? I think it might make sense to revert this again.